src/main/java/cam72cam/immersiverailroading/items/ItemSwitchKey.java

```diff
@@ -1,14 +1,33 @@
 package cam72cam.immersiverailroading.items;
 
 import cam72cam.immersiverailroading.ImmersiveRailroading;
+import cam72cam.immersiverailroading.library.ChatText;
 import cam72cam.immersiverailroading.library.GuiText;
+import cam72cam.immersiverailroading.library.Permissions;
+import cam72cam.immersiverailroading.library.SwitchState;
+import cam72cam.immersiverailroading.tile.TileRail;
+import cam72cam.immersiverailroading.tile.TileRailBase;
+import cam72cam.immersiverailroading.util.BlockUtil;
 import cam72cam.immersiverailroading.util.IRFuzzy;
+import cam72cam.mod.entity.Player;
 import cam72cam.mod.item.*;
+import cam72cam.mod.math.Vec3d;
+import cam72cam.mod.math.Vec3i;
+import cam72cam.mod.serialization.TagField;
+import cam72cam.mod.text.PlayerMessage;
+import cam72cam.mod.util.Facing;
+import cam72cam.mod.world.World;
 
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 public class ItemSwitchKey extends CustomItem {
+	public static final long CLICK_COOLDOWN_MILLIS = 250L;
+	public static final String LAST_USED_ON_KEY = "lastUsedOn";
```

Comment:
These don't really need to be their own static strings and could be inline.  Not a huge thing, just minor preference.